R/pkg/R/generics.R

```diff
@@ -735,6 +752,8 @@ setGeneric("between", function(x, bounds) { standardGeneric("between") })
 setGeneric("cast", function(x, dataType) { standardGeneric("cast") })
 
 #' @rdname columnfunctions
+#' @param x a Column object.
+#' @param ... additional argument(s).
```

Comment:
not completely sure about `contains` - is it possible to put @param in the func definition?